public/js/app/util/helpers.js

```diff
@@ -38,11 +38,11 @@ export const getDateFormat = (date, language) => {
 }
 
 export const combineMemoName = (roundInfo, semester, langAbbr = 'sv') => {
-  const { firstTuitionDate, shortName, ladokRoundId, language } = roundInfo
+  const { firstTuitionDate, shortName, language, applicationCodes } = roundInfo
   const langIndex = langAbbr === 'en' ? 0 : 1
   const { extraInfo } = i18n.messages[langIndex]
 
-  const seasonOrShortName = shortName ? shortName + ' ' : `${seasonStr(langIndex, semester)}-${ladokRoundId}`
+  const seasonOrShortName = shortName ? shortName + ' ' : `${seasonStr(langIndex, semester)}-${applicationCodes[0]}`
```

Comment:
safe extracting of applicationCodes[0]

Reply:
Changed array to direct string. Now round.applicationCode will be used.